### PR TITLE
[13.0][MIG] purchase_line_refund_to_invoice_qty

### PR DESCRIPTION
**Pull request by ChristianSantamaria**
Standard backport to v13

### CODE REVIEW

**Comment by ChristianSantamaria** [comment]
@dalonsod installed in alu enviroment test, can u test it?

**Comment by dalonsod** [comment]
Tested with a main PO CXP24/00005. Three different "little POs" belong to this main PO:
- When every "little PO" was fully received and invoiced, main PO was set to "Fully invoiced" invoice extended status.
- Then, for "little PO" CXA24/00010 I made a partial refund unchecking "quantities to be reinvoiced in PO"... both little PO and main PO remained as "Fully invoiced" :+1:  (desired new behavior)
- Then, for "little PO" CXA24/00012 made a refund checking "quantities to be reinvoiced in PO" (default Odoo behavior)... and both little PO and PO were moved to "to invoice" and "Clasif. invoice pending" respectively :+1: 
- Reinvoicing that last little PO again... we obtain "fully invoiced" status :+1: 

This works perfectly IMO.

@ChristianSantamaria could you check `es` translations? They're missing

cc @lmiguens-solvos

**Comment by dalonsod** [comment]
`i18n` stuff is also missing in v14 implementation. So it could be better adding it in a separate commit, so it could be cherry-picked to v14

File: purchase_line_refund_to_invoice_qty/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+"(comportamiento estándar)"
+
+#. module: purchase_line_refund_to_invoice_qty
+#: model_terms:ir.ui.view,arch_db:purchase_line_refund_to_invoice_qty.view_account_move_reversal
+msgid ""
+"Leave it marked when other customer invoices are expected for the quantities "
+"in the credit note."
+msgstr ""
+"Déjelo marcado cuando se esperen otras facturas de clientes por las "
+"cantidades de la nota de crédito."
```
Comment (dalonsod) [inline purchase_line_refund_to_invoice_qty/i18n/es.po]: Better this:
```suggestion
"cantidades de la factura rectificativa."
```

File: purchase_line_refund_to_invoice_qty/i18n/es.po [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+#. module: purchase_line_refund_to_invoice_qty
+#: model:ir.model.fields,field_description:purchase_line_refund_to_invoice_qty.field_purchase_order_line__qty_refunded_not_invoiceable
+msgid "Quantity Refunded Not Invoiceable"
+msgstr "Cantidad reembolsada no facturable"
+
+#. module: purchase_line_refund_to_invoice_qty
+#: model:ir.model.fields,field_description:purchase_line_refund_to_invoice_qty.field_account_move_reversal__purchase_qty_to_reinvoice
+msgid "This credit note will be reinvoiced"
+msgstr "Esta nota de crédito se refacturará"
```
Comment (dalonsod) [inline purchase_line_refund_to_invoice_qty/i18n/es.po]: The same:
```suggestion
msgstr "Esta factura rectificativa se refacturará"
```

**Review by dalonsod — APPROVED** [review]
See only minor translations changes required (based on https://github.com/OCA/account-invoicing/blob/14.0/sale_line_refund_to_invoice_qty/i18n/es.po). When done, I'll merge it.

**Pull request merged**